src/posts/helsinki-hashicorp-user-group-meetup-5.md

```diff
@@ -0,0 +1,38 @@
+---
+type: Event
+title: 'Helsinki HashiCorp User Group Meetup #5'
+subheading: Join us on 30 November at 17:00 for the fifth Helsinki HashiCorp User Group meetup!
+authors:
+- ckurowski
+tags:
+- HashiCorp
+date: 2022-28-10
+image: "/blogs/helsinki-hug-5/hug-blog-event-5.png"
+featured: false
+
+---
+
+**Helsinki HashiCorp User Group (HUG) is a quarterly meetup full of interesting talks and insights about all things HashiCorp, plus a chance to connect with fellow professionals and hobbyists alike.**
```

Comment:
It should be quarterly, but hasn't been. Should that word be dropped?

Reply:
yeah, I guess it's only twice this year. Will amend that! Still pretty active though :)